crypto/fipsmodule/ec/p384.c

```diff
@@ -256,20 +256,20 @@ static void fiat_p384_point_add(fiat_p384_felem x3, fiat_p384_felem y3,
   fiat_p384_limb_t z1nz = fiat_p384_nz(z1);
   fiat_p384_limb_t z2nz = fiat_p384_nz(z2);
 
-  // z1z1 = z1z1 = z1**2
+  // z1z1 = z1^2
```

Comment:
The notation `**` is copied from p256.c, so I prefer to keep it so it's easy to see they're the same (except for the small change in the doubling but it's not related to exponentiation).